hack/release-helper.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+set -euo pipefail
+
+
+declare -g gh_username
+declare -g gh_token
+declare -g candidate_release_tag
+declare -g release_tag
+declare -A staged_to_release=(
+    ["staged-images/kbs"]="key-broker-service"
+    ["staged-images/kbs-grpc-as"]="key-broker-service"
+    ["staged-images/rvps"]="reference-value-provider-service"
+    ["staged-images/coco-as-grpc"]="attestation-service"
+    ["staged-images/coco-as-restful"]="attestation-service"
+)
+declare -A staged_to_release_tag_prefix=(
+    ["staged-images/kbs"]="built-in-as-"
+    ["staged-images/coco-as-restful"]="rest-"
+)
+
+
+function usage_and_exit() {
+    echo
+    echo "Usage:"
+    echo "  $0  -u github-username  -k github-token  -c candidate-release-tag  -r release-tag"
+    echo
+    echo "  -u  Your github username. You'll be opening a PR against "
+    echo "      confidential-container's trustee/main."
+    echo "  -k  A github token with permissions on trustee to write packages"
+    echo "      and open PRs."
+    echo "  -c  This is the tag that's been tested and that you're happy with."
+    echo "      You want to release from this tag. Example: latest"
```

Comment:
Oh 🤔 good point. Ultimately the `candidate-release-tag` is used like so:
```
docker pull ${ghcr_repo}/${staged_pkg_name}:${candidate_release_tag}
```
E.g. for kbs:
```
docker pull ghcr.io/confidential-containers/staged-images/kbs:dc01f454264fb4350e5f69eba05683a9a1882c41
```
So it must be a tag for a GHCR package, but because we use commit shas for tagging our packages -- well -- a commit sha can be used. Let me update to make it clearer.

Reply:
Ok, adopted your language suggestion [here](https://github.com/confidential-containers/trustee/compare/49912e171c9cffa0a764f94e11c0a54d9f2c09f3..2a414e34838728d67f6602ae68e33df25a2ea403) b/c it's better, I think. Thanks!